### DISCUSSION TITLE
Highlights in specific channels

### DISCUSSION THREAD

**Discussion by chancedabs** [Chatterino/chatterino2#3123] [Ideas]
I'm not 100% sure if there isn't already a way to do this, but I was trying to make a certain text string only highlight and appear in my /mentions if it occurs in a specific channel.

I am trying to have "chance" highlighted, but I have a ton of tabs of different streams and people just say "chance" way too often, so I wanted it to only apply to a few channels where I regularly talk to people.

Not entirely important but I thought that would be handy if added!

**Comment by apa420** [comment]
You can create a filter on /mentions for the desired effect https://wiki.chatterino.com/Filters/#variables

Eg:
`((channel.name) == ("channel1") || (channel.name) == ("channel2")) && (message.content contains "chance")`

**Reply by chancedabs** [reply]
Thanks for replying, this looks like exactly what I need and the code makes perfect sense to me!

However, for some reason I'm trying and I can't get any filters to work for me. I added that code, set the filter to my /mentions tab, restarted Chatterino but nothing I try to filter shows up in /mentions

Even when I tried just pressing Quick-Add for
`message.content contains "hello"`

I test it out in chat and I can't get anything to filter into /mentions

**Reply by zneix** [reply]
@chancedabs you'll first need to add `chance` to your highlights actually for it to work, then you can filter out the /mentions split.
To be fair, this is more of a dirty hack than a solution and highlights system in general need a complete overhaul but for the time being this is all you can do I think. Hit me up if you have more questions.

**Reply by chancedabs** [reply]
Thank you so much, works perfectly!

**Comment by jupjohn** [comment]
#2882 will cover this (when it's eventually done)

**Reply by chancedabs** [reply]
Good to hear!

**Comment by Mm2PL** [comment]
Duplicates https://github.com/Chatterino/chatterino2/issues/5219
